src/shared/Atlassian.Bitbucket/BitbucketConstants.cs

```diff
@@ -44,7 +45,18 @@ public static class Credential
                 public const string DevOAuthClientId = "bitbucketDevClientId";
                 public const string DevOAuthClientSecret = "bitbucketDevClientSecret";
                 public const string DevOAuthRedirectUri = "bitbucketDevRedirectUri";
+                public const string AuthenticationModes = "bitbucketAuthModes";
             }
         }
+
+        /// <summary>
+        /// Supported authentication modes for Bitbucket.org
+        /// </summary>
+        public const AuthenticationModes DotOrgAuthenticationModes = AuthenticationModes.Basic | AuthenticationModes.OAuth; //| AuthenticationModes.Pat;
```

Comment:
If we're not adding an explicit `Pat` option (and just use the basic password field to paste a PAT in, which is perfectly OK), then please drop the commented out options. 😁